Sources/Common/Service/UserAgentUtil.swift

```diff
@@ -23,7 +23,7 @@ public class UserAgentUtilImpl: UserAgentUtil {
      * `Customer.io iOS Client/1.0.0-alpha.16`
      */
     public func getUserAgentHeaderValue() -> String {
-        var userAgent = "Customer.io iOS Client/\(deviceInfo.sdkVersion)"
+        var userAgent = "Customer.io \(deviceInfo.osName ?? "iOS" ) Client/\(deviceInfo.sdkVersion)"
```

Comment:
Lets please add test case for this, because our mix-panel tracking is checking for keyword "Customer.io iOS" and so we should make sure it doesn't break

Reply:
Good call out. I will add additional test case for `visionos`. However, this won't make the backend work for `visionos`. I have separate PRs for tha backend.